sandbox/data/cohort.json

```diff
@@ -0,0 +1,681 @@
+[
+  {
+    "authors": "[\"Author name\"]",
```

Comment:
Is this field manipulated as an array of strings or just a string?

In Cohort API this is a string and is being displayed weird on front-end `[\"Author name\"]`. We might wanna consider just leaving `Author name`

Reply:
This is the data that's being pulled out of the published-cohort-definitions API currently, so if we were going to change it here, we'd have to change it there as well. I'd suggest that we create a ticket for sorting it out there, with a note on it saying that it'll need changing here as well, and I'll put a comment into the handover doc. But we leave it as is here for now.

Reply:
Thanks Rob - I'm happy with a new ticket to address this issue. 👍

Reply:
New ticket https://nhsd-jira.digital.nhs.uk/browse/CAAS-1753. Needs a BA review, but it's in the backlog now.